dojo/finding/views.py

```diff
@@ -804,7 +804,14 @@ def edit_finding(request, fid):
             # any existing finding should be updated
             push_to_jira = push_to_jira and not push_group_to_jira and not new_finding.has_jira_issue
 
-            new_finding.save(push_to_jira=push_to_jira)
+            # if we're removing the "duplicate" in the edit finding screen
+            # do not relaunch deduplication, otherwise, it's never taken into account
+            if old_finding.duplicate and not new_finding.duplicate:
+                reset_finding_duplicate_status_internal(request.user, new_finding.id)
```

Comment:
I wouldn't use this function here as it leads to weird code with refresh from db. Better just set duplicate=False, duplicat_finding=None, active=True and save without dedupe

Reply:
OK.

I don't think I want to even force it to `Active`, but I don't think we have anything better right now. We should somehow be able to tell the last status prior to dedupe (e.g. what status it was imported in) and reset to that.